examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -478,8 +481,8 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
           case "done":
             break;
           default:
-            console.error("ws get unknown data: ", result);
-            cloudberryService.errorMessage = "ws get unknown data: " + result.toString();
+            console.log("ws get unknown data: ", result);
+            //cloudberryService.errorMessage = "ws get unknown data: " + result.toString();
```

Comment:
Let's remove this comment. We can add it later once we fix the issue.

Reply:
Fixed